ICAFIX/hcp_fix

```diff
@@ -378,4 +393,16 @@ log_Msg "Done renaming files"
 # Remove the 'fake-NIFTI' file created in fix_3_clean for high-pass filtering of the CIFTI (if it exists)
 $FSLDIR/bin/imrm ${fmrihp}.ica/Atlas
 
+#delete things with too-generic names
+rm -f ${fmrihp}.ica/filtered_func_data.nii.gz ${fmrihp}.ica/Atlas.dtseries.nii
+
+#optional cleanup of hp intermediates
+if [[ "$DeleteIntermediates" == "TRUE" ]]
+then
+    rm -f ${fmrihp}.ica/Atlas_hp_preclean.dtseries.nii ${fmri}_hp${HighPass}.nii.gz
```

Comment:
This script does not have a `${HighPass}` variable.  It is called `${hp}`, or use `${hpStr}` defined at L367.  Regardless, removal of the .nii.gz needs to be conditioned on hp>=0.

Reply:
I don't know how this got through testing, as I tested with hp 2000 and looked for the file in question after running it, and didn't find it.  Should be fixed by 9aea92921075857b7f6041ef9d2b6b93ee6c8698.